cmd/create/oidcprovider/cmd.go

```diff
@@ -226,14 +226,18 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 }
 
-func createProvider(r *rosa.Runtime, oidcEndpointUrl string, clusterId string) error {
-	thumbprint, err := oidcconfigs.FetchThumbprint(oidcEndpointUrl)
```

Comment:
I found this func() is still used I manual mode, should we also change this path?

Reply:
Thanks, fixed